packages/core/src/components/Search/__stories__/Search.mdx

```diff
@@ -53,6 +69,53 @@ There are three sizes available: Small (32px), Medium (40px), and Large (48px).
 
 <Canvas of={SearchStories.Sizes} />
 
+### With additional action
+
+The search field can contain one more search-related action (like filtering). This action should be represented by an icon button on the right.
+
+<Canvas of={SearchStories.WithAdditionalAction} />
+
+### With clear button
+
+Once the user starts typing, include a clear button that lets the user delete their current search quickly.
+
+<Canvas of={SearchStories.WithClearButton} />
+
+## Do's and Don'ts
+
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component: <Search value="User typing" wrapperClassName={styles.fixedWidth} />,
+        description: "Keep the search icon on the left side of the input, and the clear button on the right side."
+      },
+      negative: {
+        component: (
+          <Search
+            value="User typing"
+            searchIconName={CloseSmall}
+            clearIconName={SearchIcon}
+            wrapperClassName={styles.fixedWidth}
+          />
+        ),
+        description: "Don't place the search icon on the right side of the input, or the clear button on the left side."
+      }
+    },
+    {
+      positive: {
+        component: <Search value="User typin" wrapperClassName={styles.fixedWidth} />,
```

Comment:
typin?

Reply:
yep on purpose, the "don't" is with "typing"

Reply:
It looks like a typo haha, maybe add `...` like in whatsapp to indicate that typing is still in progress